.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish Release
+on:
+  push:
+    branches: [master, main]
+    tags: ["*"]
+jobs:
+  publish:
+    name: Publish to Sonatype Releases
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: olafurpg/setup-scala@v13
+        with:
+          java-version: openjdk@1.17
+      - uses: coursier/cache-action@v6.3
+      - run: sudo apt update && sudo apt install -y gnupg
+      - run: echo $PGP_SECRET | base64 --decode | gpg --batch --import
+        env:
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+      - run: sbt +test ciReleaseTagNextVersion ciReleaseSonatype
+        env:
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```

Comment:
Who owns these? I'm concerned that these might be lost somewhere when one of the dev's leave.